tests/baselines/reference/goToDefinitionSwitchCase1.baseline.jsonc

```diff
@@ -0,0 +1,17 @@
+// === goToDefinition ===
+// === /tests/cases/fourslash/goToDefinitionSwitchCase1.ts ===
+// [|{| contextId: 0 |}switch|] (<|null|>) {
```

Comment:
i think contextSpan should be `switch (null)` instead of  just `null`

Reply:
Was the previous commit correct then?

Reply:
kind of.. probably needs closing paren as well part of context ?